webpack rtl.scss Error

ERROR in ./src/content/scss/vendor.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/src!./node_modules/sass-loader/dist/cjs.js??ref--11-3!./src/content/scss/vendor.scss)
         Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
         SassError: Can't find stylesheet to import.
            ╷
         12 │ @import 'src/Dotnetcoresample/ClientApp/src/content/scss/rtl.scss';
            │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
            ╵
           /Users/developer/Desktop/dotnetcoresample/src/Dotnetcoresample/ClientApp/src/content/scss/vendor.scss 12:9  root stylesheet
         npm ERR! code ELIFECYCLE
         npm ERR! errno 2
         npm ERR! dotnetcoresample@0.0.1-SNAPSHOT webpack: `node --max_old_space_size=4096 node_modules/webpack/bin/webpack.js "--config" "webpack/webpack.dev.js" "--env.stats=minimal"`
         npm ERR! Exit status 2
         npm ERR! 




##### **JHipster Version(s)**

```
/Users/developer
└── generator-jhipster@6.10.1 

```


##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**


<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "nativeLanguage": "en"
    },
    "namespace": "DanaIns",
    "databaseType": "sqllite",
    "authenticationType": "jwt",
    "prodDatabaseType": "mysql",
    "jhipsterVersion": "6.9.1",
    "applicationType": "monolith",
    "baseName": "dotnetcoresample",
    "useSass": true,
    "clientPackageManager": "npm",
    "clientFramework": "angularX",
    "clientTheme": "none",
    "clientThemeVariant": "",
    "creationTimestamp": 1594448870275,
    "testFrameworks": ["gatling"],
    "jhiPrefix": "jhi",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": [
      {
        "name": "generator-jhipster-dotnetcore",
        "version": "1.0.0"
      }
    ],
    "enableTranslation": true,
    "nativeLanguage": "en",
    "languages": ["en", "fa"],
    "blueprints": [
      {
        "name": "generator-jhipster-dotnetcore",
        "version": "1.0.0"
      }
    ]
  }
}

</pre>
</details>


##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>


##### **Environment and Tools**

openjdk version "1.8.0_242"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_242-b08)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.242-b08, mixed mode)

git version 2.20.1

node: v13.7.0

npm: 6.14.4

yeoman: 3.1.1

yarn: 1.21.1

Docker version 19.03.8, build afacb8b

docker-compose version 1.25.4, build 8d51620a

--- a/generators/client/files-common.js
+++ b/generators/client/files-common.js
@@ -188,6 +188,15 @@ function updateTestFramework(){
     }
 }
 
+function updateVendor() {
+   this.replaceContent(
+        `${SERVER_SRC_DIR}${this.mainClientAppDir}/content/scss/vendor.scss`,
+        `${SERVER_SRC_DIR}${this.mainClientDir}/src/content`,
+        "..",
+        true
+    );
+}
+
 function writeFiles() {
     updateWebpackCommonJs.call(this);
     updateWebpackDevJs.call(this);
@@ -198,6 +207,7 @@ function writeFiles() {
     updateJestConf.call(this);
     updateEsLinIgnore.call(this);
     updateTestFramework.call(this);
+    updateVendor.call(this); 
 }
 
 module.exports = {

--- a/generators/languages/index.js
+++ b/generators/languages/index.js
@@ -141,7 +141,7 @@ module.exports = class extends LanguageGenerator {
     _updateLanguagesInLanguagePipeDotNetCore(languages) {
         const fullPath =
             this.clientFramework === 'angularX'
-                ? `${constants.SERVER_SRC_DIR}${this.mainClientAppDir}/shared/language/find-language-from-key.pipe.ts`
+                ? `${constants.SERVER_SRC_DIR}${this.mainClientAppDir}/app/shared/language/find-language-from-key.pipe.ts`
                 : `${constants.SERVER_SRC_DIR}${this.mainClientDir}/app/config/translation.ts`;
         try {
             let content = '{\n';
@@ -174,7 +174,7 @@ module.exports = class extends LanguageGenerator {
         if (this.clientFramework !== 'angularX') {
             return;
         }
-        const fullPath = `${constants.SERVER_SRC_DIR}${this.mainClientAppDir}/core/language/language.constants.ts`;
+        const fullPath = `${constants.SERVER_SRC_DIR}${this.mainClientAppDir}/app/core/language/language.constants.ts`;
         try {
             let content = 'export const LANGUAGES: string[] = [\n';
             languages.forEach((language, i) => {

